cartoframes/auth/__init__.py

```diff
@@ -1,98 +1,130 @@
 
         .. code::
 
-            from cartoframes.auth import Context, set_default_context
+            from cartoframes.auth import set_default_credentials
 
-            context = Context(
+            set_default_credentials(
                 base_url='https://your_user_name.carto.com',
                 api_key='your api key'
             )
-            set_default_context(context)
 
-        From credentials.
+            # or
+
+            set_default_credentials(
+                'https://your_user_name.carto.com',
+                'your api key'
+            )
+
+        From a base_url (for public datasets).
+        The API key `default_public` is used by default.
+
+        .. code::
+
+            from cartoframes.auth import set_default_credentials
+
+            set_default_credentials('https://your_user_name.carto.com')
+
+        From a :py:class:`Credentials <cartoframes.auth.Credentials>` class.
 
         .. code::
 
-            from cartoframes.auth import Credentials, set_default_context
+            from cartoframes.auth import Credentials, set_default_credentials
 
-            creds = Credentials(
-                api_key='abcdefg',
-                username='cartoframes'
+            credentials = Credentials(
+                base_url='https://your_user_name.carto.com',
+                api_key='your api key'
             )
-            set_default_context(creds)
+            set_default_credentials(credentials)
     """
-    global _default_context
+    global _default_credentials
 
-    _context = context if first is None else first
     _base_url = base_url if first is None else first
+    _username = username if first is None else first
     _api_key = (api_key if second is None else second) or 'default_public'
-    _creds = creds if first is None else first
-
-    if isinstance(_context, Context):
-        _default_context = _context
-    elif isinstance(_base_url, str) and isinstance(_api_key, str):
-        _default_context = Context(base_url=_base_url, api_key=_api_key, session=session)
-    elif isinstance(_creds, Credentials):
-        _default_context = Context(creds=_creds, session=session)
+    _credentials = credentials if first is None else first
+
+    if isinstance(_credentials, Credentials):
+        _default_credentials = _credentials
+
+    elif isinstance(_base_url or _username, str) and isinstance(_api_key, str):
+        if is_url(_base_url):
+            _default_credentials = Credentials(base_url=_base_url, api_key=_api_key)
+        else:
+            _default_credentials = Credentials(username=_username, api_key=_api_key)
+
     else:
-        raise ValueError('Invalid inputs. Pass a Context, a base_url and api_key pair, or a Credentials object.')
+        raise ValueError(
+            'Invalid inputs. Pass a Credentials object, a username and api_key pair or a base_url and api_key pair.')
+
+    if session:
+        _default_credentials.session = session
 
 
 __all__ = [
     'Context',
```

Comment:
Is 'Context' going to be removed here?

Reply:
Yes, now this is still in the file to keep the tests working, but it will be removed in the next PR.